nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-workflow/src/main/resources/OSGI-INF/notification-contrib.xml

```diff
@@ -24,6 +24,3 @@
   </extension>
 
 </component>
-
-
-
```

Comment:
i think this file is not related to this jira ticket after the cleanup there is no code changes or i am missed some thing :)

Reply:
notification-contrib.xml will enable autosubscribe by default from now on as you will see in next commit.
Also, This means a lot of tests will break because of wrong mail count. So I had to disable autosubscribe with the CommentFeature. 
So the notification-contrib.xml of the tests which disables them has been added to it.
Then I have my test contrib which enables them just for the notification autosubscribe test.

Reply:
i think if we don't make changes on a given file we shouldn't reformat/clean it even we import the contribution

Reply:
My bad, I edited the wrong file.